src/main/java/input/file/PlainTextFile.java

```diff
@@ -0,0 +1,50 @@
+package input.file;
+
+import java.io.ByteArrayInputStream;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+
+/**
+ * Class to specify a .txt file.
+ * @author Paul
+ *
+ */
+public class PlainTextFile extends DataFile {
+	
+	public PlainTextFile(String path) {
+		super(path);
+	}
+	
+	@Override
+	public InputStream getDataStream() throws FileNotFoundException {
+		InputStream stream = new FileInputStream(getFile());
+		Scanner scanner = new Scanner(stream, "UTF-8");
+		scanner.useDelimiter("\\A");
+		StringBuilder builder = new StringBuilder();
```

Comment:
Kan je niet op een of andere manier de builder een goeie begin capaciteit geven, anders moet hij om de haverklap zijn buffer vergroten en dan ben je alsnog nergens.